CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 0.37.0
+
+- We have added a new optional parameter namespace to fork_create method on git projects, which allows you to
+  fork project into a specific namespace. (Forking to namespaces is not allowed on Pagure.) (#685)
+- We have implemented get_contributors function that can be used for getting contributors to the project on GitHub
+  (set of logins) and GitLab (set of authors). (#692)
+- We have introduced a new exception class GitForgeInternalError that indicates failure that happened within the forge
+  (indicated via 50x status code). \*APIException have been given a new superclass APIException that provides status
+  code from forge (in case of error, invalid operation, etc.). (#690)
```

Comment:
```suggestion
- We have introduced a new exception class `GitForgeInternalError` that indicates failure that happened within the forge
  (indicated via 50x status code). `\*APIException` have been given a new superclass `APIException` that provides status
  code from forge (in case of error, invalid operation, etc.). (#690)
```